t/redirects.rb

```diff
@@ -0,0 +1,10 @@
+require 'test_helper'
+
+describe 'redirecting links' do
+  around { |test| VCR.use_cassette('liff', &test) }
+  subject { WikiData.ids_from_pages('en', ['The Meaning of Liff', 'The Deeper Meaning of Liff']) }
```

Comment:
I don't mind if you do things in separate PRs, but I don't think this could go live like this.